queries/pod/highlights.scm

```diff
@@ -0,0 +1,40 @@
+; A highlight file for nvim-treesitter to use
+
+[(pod_directive)
+ (head_directive)
+ (over_directive)
+ (item_directive)
+ (back_directive)
+ (encoding_directive)
+ (cut_directive)] @keyword
+
+(head_paragraph (content) @string)
+
+(over_paragraph (content) @string)
+(item_paragraph (content) @string)
+(encoding_paragraph (content) @string)
+
+(verbatim_paragraph (content) @verbatim)
+
+(interior_sequence) @variable
+
+(interior_sequence
+  (sequence_letter) @letter
+  (#match? @letter "B")
```

Comment:
```suggestion
  (#match? @letter "B")
```
prefer `#eq?`, `#any-of?`, and `#lua-match?` (in that order)

Reply:
I assume `#eq?` can't be used because it might have multiple letters. But `#lua-match?` can.

Reply:
`#eq?` would be fine, I think I just wrote this very early before I realised that `#eq?` even existed; having only seen `#match?` elsewhere